app/views/orders/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<p style="color: green"><%= notice %></p>
+
+<h1>Orders</h1>
+
+<div id="orders" class="container-fluid d-flex gap-2">
+  <% @orders.each do |order| %>
+    <%= render order %>
+  <% end %>
+</div>
```

Comment:
I'm not sure that [Step 8](https://github.com/IrynaKolh/R7-additional/blob/main/lessons/Assignment-11-Styling-Rails-With-Bootstrap.md#step-8-showing-orders-as-cards-in-a-grid) was followed here, because when I shrink my browser, all of the cards become squished like so:

![Screenshot 2024-09-07 at 2 07 48 AM](https://github.com/user-attachments/assets/1288c068-f2a3-40f0-ba5d-e4f8c1ea3690)

What it should look like when I have a browser that small:

![Screenshot 2024-09-07 at 2 10 07 AM](https://github.com/user-attachments/assets/0ab97a13-b7d9-471d-9cd1-f9e0c5fd86d6)

And what it should look like if I enlarge it just enough to fit two cards:

![Screenshot 2024-09-07 at 2 09 49 AM](https://github.com/user-attachments/assets/2835eec2-529f-4eeb-9132-2c65572d1e15)

This kind of adjustment according to the browser width is what makes it "responsive." Squishing the cards is not a responsive design.

Reply:
I think easiest way is a use: flex-wrap

![image](https://github.com/user-attachments/assets/4709d46e-e314-4db4-ae7c-f0a3b5e346fd)

Reply:
flex-wrap has a similar effect, but it gives you less control over the columns. This lesson doesn't go into detail about it (our course focuses on Rails, not CSS), so one could argue that perhaps it's simpler to just use flex-wrap going forward, but just know that grids can be more complicated; for example: <https://getbootstrap.com/docs/5.3/layout/grid/#mix-and-match>